### PR TITLE
fix(GiniBankSDK):

### PR DESCRIPTION
**Pull request by zladzeyka**
Remove "Save functionality" when the back button is clicked.

PIA-2810

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/LineItemDetailsViewController.swift [hunk 1]
```diff
@@ -79,13 +79,6 @@ class LineItemDetailsViewController: UIViewController {
         super.viewWillDisappear(animated)
         guard let lineItem = lineItem, !lineItem.isUserInitiated else { return }
```
Comment (a-szotyori) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/LineItemDetailsViewController.swift:80]: This line has no use anymore. There method is empty after the guard so there is nothing to guard...

The whole viewWillDisappear method has no use anymore and can be deleted.

**Review by a-szotyori — CHANGES_REQUESTED** [review]
You can delete more code. 😉

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/LineItemDetailsViewController.swift (hunk 1 above)
Comment (zladzeyka) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/LineItemDetailsViewController.swift]: Good catch 😉

**Review by a-szotyori — APPROVED** [review]

**Review by chriskryn — APPROVED** [review]
:+1:

**Pull request merged**